### ISSUE TITLE
Use Strict Warning

### ISSUE THREAD

**Issue by hnordt** [SplittyDev/evee.js#4]
Hello!

Does `'use strict';` is really necessary?

Here's the warning I'm receiving for a `create-react-app` project:

```
./src/smalldots/~/evee/dist/evee.js

/Users/hnordt/workspace/genesis/src/smalldots/node_modules/evee/dist/evee.js
  1:1  warning  'use strict' is unnecessary inside of modules  strict

✖ 1 problem (0 errors, 1 warning)
```

**Comment by SplittyDev** [comment]
@hnordt thanks for pointing that out.
It shouldn't be necessary, since ES6 enforces strict mode by default.

I'll fix that within the next few minutes.

**Comment by hnordt** [comment]
Thank you! That's the only missing part for my next release:

https://github.com/smalldots/smalldots/commit/ed2f12b2ab255c3fa22c2a0ca5e690ef8053775e

**Comment by SplittyDev** [comment]
@hnordt fixed it, version 1.1.1 of the package contains the fixes.

I feel honored that awesome people like you use evee to build amazing software,
so thank you again for your help :)

People like you motivate me to continue writing open-source software every day.

**Comment by hnordt** [comment]
Thank you @SplittyDev, your library is amazing too 👍
